### ISSUE TITLE
Adding a Line in Mobile View Generates Two Control Points

### ISSUE THREAD

**Issue by LDSmj** [excalidraw/excalidraw#8617]
When adding a line in mobile view, two control points are generated even with a single tap.

https://github.com/user-attachments/assets/6e714d0a-e7cf-4ff8-b12c-ffccf12b668b

**Comment by beingPro007** [comment]
@dwelle can i work on this i  want to try this one

**Comment by beingPro007** [comment]
https://github.com/user-attachments/assets/2e6658b2-27a2-43c3-90cf-a7443c28524b
do u want this things in fone

**Comment by LDSmj** [comment]
I don't need three control points; rather, I want to avoid generating two control points with a single tap. In mobile view, to continue drawing a line, you must first tap and then drag to where you want the line to extend. The problem is that with a single tap, it generates two control points, which disrupts the design I want to create.


https://github.com/user-attachments/assets/088f10d9-b15d-45e8-afdc-403c3542ad30

**Comment by beingPro007** [comment]
> I don't need three control points; rather, I want to avoid generating two control points with a single tap. In mobile view, to continue drawing a line, you must first tap and then drag to where you want the line to extend. The problem is that with a single tap, it generates two control points, which disrupts the design I want to create.
> 
>  Excalidraw.-.line-bug.mp4

ok that's the point let me see what's happening and try to resolve it
Thank you for the video explanations

**Comment by LDSmj** [comment]
> > I don't need three control points; rather, I want to avoid generating two control points with a single tap. In mobile view, to continue drawing a line, you must first tap and then drag to where you want the line to extend. The problem is that with a single tap, it generates two control points, which disrupts the design I want to create.
> > Excalidraw.-.line-bug.mp4
> 
> ok that's the point let me see what's happening and try to resolve it Thank you for the video explanations

Thank you for your help! I hope the issue gets resolved soon.

**Comment by beingPro007** [comment]
Yes right now I'm working on it if solved I let you know about this

**Comment by LDSmj** [comment]
Hi, I just want to follow up to see if there are any updates on this bug?

**Comment by LDSmj** [comment]
Hi, I just wanted to follow up and see if there are any updates on this issue. It has been a few months since it was posted, and the problem still persists. Any updates or assistance would be greatly appreciated. Thank you!
